rtmbot.py

```diff
@@ -145,14 +147,42 @@ def main_loop():
 
 def parse_args():
     parser = ArgumentParser()
-    parser.add_argument(
+
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
         '-c',
         '--config',
         help='Full path to config file.',
         metavar='path'
     )
+    group.add_argument(
+        '-s',
+        '--settings',
+        help='Python path to settings module.',
+        metavar='module'
+    )
+
+    # Allows the bot to be run with zero arguments and use a Python module
```

Comment:
Since the group is mutually exclusive, we cannot set the defaults for both, or invoking the bot will be buggy. So, look for an environment variable named `TEAMBOT_SETTINGS_MODULE` to decide what to do.